bsp/worker/src/mill/bsp/worker/MillBuildServer.scala

```diff
@@ -45,21 +45,98 @@ private class MillBuildServer(
   // progresses through:
   //
   // Set when the client connects
-  protected var client: BuildClient = scala.compiletime.uninitialized
+  protected var client: BuildClient = null
```

Comment:
Why is `uninitialized` not enough? Can we use a better type then, e.g. `Option`?